lighthouse-core/report/html/report-styles.css

```diff
@@ -102,6 +103,7 @@
   --env-item-bg: var(--color-black-100);
   --env-name-min-width: 220px;
   --env-tem-padding: 10px 0px;
+  --expandable-padding: 0 0 2px 28px;
```

Comment:
magic number here...  :/

```
    --score-shape-margin: 7px 12px 0 4px;
    --score-shape-size: 12px;
```

afaik the 28 is 12 + 4 + 12.  

I wonder if we can redefine score-shape with some base variable so these remain synchronized?

Reply:
done-ish. shoot, that icon width varies based on the state ...